packages/client/components/ActivityLibrary/ActivityDetails/TemplateDetails.tsx

```diff
@@ -73,6 +73,22 @@ const ACTIVITY_TYPE_DATA_LOOKUP: Record<
   }
 }
 
+const ACTIVITY_ID_DATA_LOOKUP: Record<
+  string,
+  {description: React.ReactNode; integrationsTip: React.ReactNode}
+> = {
+  oneOnOneAction: {
+    description: (
+      <>
+        This is a space to check in one-on-one. Share a personal update using the <b>Icebreaker</b>{' '}
+        phase. Give a brief update on what’s changed with your work during the <b>Solo Updates</b>{' '}
+        phase. Raise issues for discussion in the <b>Agenda</b> phase.
```

Comment:
+1 Is the intention to continue including solo updates in 1:1s? Most 1:1 meetings just follow a single agenda.

Reply:
For now we will have solo updates. Later we will add an option to disable this step